cluster/src/main/java/com/hartwig/pipeline/tertiary/virus/VirusAnalysis.java

```diff
@@ -10,27 +10,29 @@
 import com.hartwig.pipeline.execution.vm.BashCommand;
 import com.hartwig.pipeline.execution.vm.BashStartupScript;
 import com.hartwig.pipeline.execution.vm.VirtualMachineJobDefinition;
-import com.hartwig.pipeline.execution.vm.VmDirectories;
-import com.hartwig.pipeline.execution.vm.unix.ExportPathCommand;
 import com.hartwig.pipeline.metadata.AddDatatype;
 import com.hartwig.pipeline.metadata.ArchivePath;
 import com.hartwig.pipeline.metadata.SomaticRunMetadata;
 import com.hartwig.pipeline.report.Folder;
 import com.hartwig.pipeline.report.RunLogComponent;
 import com.hartwig.pipeline.report.SingleFileComponent;
 import com.hartwig.pipeline.resource.ResourceFiles;
+import com.hartwig.pipeline.stages.SubStageInputOutput;
 import com.hartwig.pipeline.storage.GoogleStorageLocation;
 import com.hartwig.pipeline.storage.RuntimeBucket;
 import com.hartwig.pipeline.tertiary.TertiaryStage;
-import com.hartwig.pipeline.tools.Versions;
 
-public class VirusBreakend extends TertiaryStage<VirusBreakendOutput> {
+public class VirusAnalysis extends TertiaryStage<VirusOutput> {
 
     public static final String NAMESPACE = "virusbreakend";
 
+    public static final String VIRUS_BREAKEND_VCF = ".virusbreakend.vcf";
+    public static final String VIRUS_BREAKEND_SUMMARY = ".virusbreakend.vcf.summary.tsv";
```

Comment:
There is a way we can remove this duplication between the constructor of the VirusInterpreter substage and this constant by storing the results in a field. See unfilteredVcf in StructuralCaller for an example.

Downside is we end up with a non-final field... I'll leave it to you which evil you want to trade off :)

Reply:
Will leave as-is. I think a bit more dodgy part of this code is that for virus interpreter I specify an output dir and pipeline has to assume a specific file is written. Would have been cleaner to explicitly write to the file the pipeline needs. Will also leave as-is, don't think this will change a lot anyway or is specifically brittle